client/src/App.js

```diff
@@ -1,24 +1,51 @@
-import React from "react";
-import logo from "./logo.svg";
-import "./App.css";
+import React, { useState, useEffect } from "react";
+import { Credentials } from './spotify/creds.js'
+import axios from 'axios'
+import Dropdown from './Components/Dropdown.js'
 
-function App() {
-  const [data, setData] = React.useState(null);
 
-  React.useEffect(() => {
-    fetch("/api")
-      .then((res) => res.json())
-      .then((data) => setData(data.message));
-  }, []);
+const App = () => {
+
+  const spotify = Credentials()
+
+  const [token, setToken] = useState('')
+  const [genres, setGenres] = useState({selectedGenre: '', listOfGenresFromAPI: []})
+
+  useEffect(() => {
+
+    axios('https://accounts.spotify.com/api/token', {
```

Comment:
ideally none of this logic lives on the frontend but rather in an express controller behind `/api`.